tests/simplex/simplex_grids.h

```diff
@@ -106,25 +106,121 @@ namespace dealii
 
     template <int dim, int spacedim>
     void
-    subdivided_hyper_cube_with_wedges(Triangulation<dim, spacedim> &tria,
-                                      const unsigned int            repetitions,
-                                      const double                  p1 = 0.0,
-                                      const double                  p2 = 1.0,
```

Comment:
As the failing test in the CI shows, we still need this function in `simplex/matrix_free_01`.